ts/Core/Series/Point.ts

```diff
@@ -458,18 +465,18 @@ class Point {
             }
 
             // Remove properties after animation
-            if (!dataSorting?.enabled) {
-                destroyPoint();
-
+            if (duration) {
+                series.condemnedPoints.push(this);
+                this.graphic?.addClass('highcharts-point-condemned');
+                setTimeout(destroyPoint, duration);
```

Comment:
Just curious, why changing `syncTimeout` to `setTimeout`?

Reply:
I mean, with duration=0, do we expect to run `destroyPoint()` right after or, do we want to wait until other operations are done?